docs/apm/traces/working-with-tracing-data/service-map.md

```diff
@@ -1,15 +1,25 @@
 ---
 id: service-map
-title: Service Map
-sidebar_label: Service Map
+title: Services List and Map
 description: View your application topology and investigate microservice interactions.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-The Service Map is a high-level view of your application environment, automatically built from tracing data in real time as it arrives in Sumo Logic. This gives you a greater understanding of your application architecture and dependencies between monitored microservices.
+To open Services List and Map, click **+ New** > **Service Map**.<br/> <img src={useBaseUrl('img/traces/open-service-map.png')} alt="open servicemap" width="400"/>
 
-To open Service Map, click **+ New** > **Service Map**.<br/> <img src={useBaseUrl('img/traces/open-service-map.png')} alt="open servicemap" width="225"/>
+## Service List view
+
+The **Service List** view provides a compact, high-level overview of your application service health insights and KPIs.
```

Comment:
...
KPIs in columns cover last 15min of data. Anomalies can be configured in Settings dialog on the right.
Filters list on the left allows you to filter the list.
You can enable disable columns using Display settings dialog on the right.
Clicking on any service opens Entities panel with possibility for further drill-downs.